### ISSUE TITLE
Support for Twine strings format

### ISSUE THREAD

**Issue by rtsisyk** [WeblateOrg/weblate#6741]
### Describe the problem

My project uses a tool called [twine](https://github.com/scelis/twine). This tool uses ini-style text file as a source for localizations:

```
[[Strings]]
  [back]                                    <!-- string id
    en = Back                               <!-- source english string
    tags = android                          <!-- tag, this string is for Android only
    comment = Button text (should be short) <!-- a comment to understand context
    af = Back
    ar = رجوع
    be = Back
    bg = Назад
    cs = Zpět
    da = Tilbage
    de = Zurück
    el = Πίσω
    en-GB = Back
    es = Atrás
```

A tool generates localization for Android and iOS by using translated strings from the file:

```
twine generate-all-localization-files --format apple data/strings/strings.txt iphone/Maps/LocalizedStrings/
twine generate-all-localization-files --format apple-plural data/strings/strings.txt iphone/Maps/LocalizedStrings/
twine generate-all-localization-files  --format android data/strings/strings.txt android/res/
```

Generated files don't lose metadata information from the source text file.
For example, android/res/values/strings.xml:

### Describe the solution you'd like

Implement support for this format.

### Describe alternatives you've considered

See discussion in https://github.com/WeblateOrg/weblate/issues/6346

We tried "2. Drop Twine and upload Android and iOS files directly", but found this approach uncomfortable. 

### Additional context

https://github.com/WeblateOrg/weblate/issues/6346

**Comment by nijel** [comment]
* Weblate currently does not support multi-language translation files, it requires each language to be in a separate file.
* Before adding a support in Weblate, translate-toolkit support for the format should be implemented, so that Weblate can build on top of that.
* By using Twine, you apparently lose some features of the native formats, such as plurals, see https://github.com/scelis/twine/issues/46

**Comment by rtsisyk** [comment]
> Weblate currently does not support multi-language translation files, it requires each language to be in a separate file.

OK 

> Before adding a support in Weblate, translate-toolkit support for the format should be implemented, so that Weblate can build on top of that.

OK 

> By using Twine, you apparently lose some features of the native formats, such as plurals, see Support plural forms scelis/twine#46

Good point.

**Comment by nijel** [comment]
An alternative approach might be to use Weblate instead of Twine - stick with only one of the native formats and download the other one using an API (or manually, for example at https://hosted.weblate.org/projects/organicmaps/android/cs/#download).

**Comment by nijel** [comment]
Yes, but before jumping on that, please consult the approach to take in Weblate – it is currently built around the expectation that every language has the own file (or set of files).

**Comment by nijel** [comment]
I've created a tracker issue for multi-language files support:
* https://github.com/WeblateOrg/weblate/issues/11039
